README.md

```diff
@@ -152,6 +222,15 @@ The script accepts the following parameters as environment variables:
 - LOG_TO_FILE: A flag indicating if the logs should be sent to a file. Optional. Default value: true.
 - LOG_DIR: The path to a directory where the log file will be searched for or created if it does not exist. Optional. Default value: "./log".
 - LOG_FILE_NAME: The name of the file where the logs will be stored. Optional. Default value: "sth_app.log".
+- SERVICE_PREFIX: The prefix to be added to the service for the creation of the databases. More information below. Optional. Default value: "sth".
```

Comment:
Fixed in https://github.com/telefonicaid/IoT-STH/commit/4a37bf6666339a8e7d137ba3cb62fca558080ab0 Thanks @frbattid ! ;)